federation/pkg/federation-controller/service/servicecontroller.go

```diff
@@ -71,8 +71,6 @@ const (
 	doNotRetry = time.Duration(0)
 
 	UserAgentName = "federation-service-controller"
-	KubeAPIQPS    = 20.0
-	KubeAPIBurst  = 30
```

Comment:
[here](https://github.com/kubernetes/kubernetes/blob/master/federation/pkg/federation-controller/cluster/cluster_client.go#L37-L38) also has the same unused const.